WPFClient/MessageType.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WPFClient
+{
+    public enum MessageType
```

Comment:
This likely won't be needed, as with SignalR the service can invoke methods on the HubConnection, so no need to differentiate between request and response messages.